README.md

```diff
@@ -6,7 +6,7 @@
 
 # kit
 
-*[DEPRECATED] Thanks for using these packages, but we will no longer support them*
+**[DEPRECATED] Thanks for using these packages, but we will no longer support them.**
```

Comment:
![iu](https://user-images.githubusercontent.com/492166/55404740-f3e15c00-552e-11e9-8abf-d72b51491a08.gif)

Reply:
Literally the _perfect_ gif.